kartothek/serialization/_parquet.py

```diff
@@ -195,6 +163,94 @@ def store(self, store, key_prefix, df):
         return key
 
 
+def _restore_dataframe(
```

Comment:
just a note regarding git diffs and git blame:

git has a much easier time figuring the history out if you actually do not change too much. Instead, you're moving this entire code section, i.e. we're effectively lossing easier git blame.
This is mostly a style note but I would prefer to define this as a static_method as well to keep diffs small

Reply:
Made `_restore_dataframe` into static method, `restore_dataframe` into class method